lib/src/bindings/bindings_player_web.dart

```diff
@@ -420,79 +428,182 @@ class FlutterSoLoudWeb extends FlutterSoLoud {
     return wasmSetMaxActiveVoiceCount(maxVoiceCount);
   }
 
+  /////////////////////////////////////////
+  /// voice groups
+  /////////////////////////////////////////
+
+  @override
+  SoundHandle createVoiceGroup() {
+    /// The group handle returned has the sign bit flagged. Since on the web
+    /// the int is a signed 32 bit, a negative number will be returned.
+    /// Fixing by ORing the result.<
+    final ret = wasmCreateVoiceGroup() | 0xfffff000;
```

Comment:
This is a bit scary: can this become a source of bugs when some internal mechanism changes? I'd prefer to at least log the original return value and the fact that we're changing it. So that if we ever get a bug from this, we at least have a way to find the problem in the logs.

Reply:
A negative number is always returned on Web and drove me crazy for many half hours! :)
Now, I don't remember well what were under the hood of JS and C++/WASM, but I tested a lot and anyway I think we will never touch the internal SoLoud C++ groups.
After saying this, there are no problems for me to log the original `wasmCreateVoiceGroup()` return value if you want.